### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by KennyKvn001**

### CODE REVIEW

**Review by dusengepeggy — CHANGES_REQUESTED** [review]
change the file name to contact also

**Review by dusengepeggy — APPROVED** [review]
LGTM

File: contact.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<!DOCTYPE html>
 <html lang="en">
   <head>
-    <title>Menu</title>
+    <title>Contact</title>
```
Comment (devark28) [inline contact.html:4]: add an html comment saying `the contacts page`

**Review by devark28 — CHANGES_REQUESTED** [review]

File: contact.html (hunk 1 above)
Comment (KennyKvn001) [reply contact.html:4]: Done

**Review by devark28 — APPROVED** [review]
Awesome work

**Pull request merged**